modules/mcuboot/CMakeLists.txt

```diff
@@ -707,6 +707,57 @@ CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT cannot be used with CONFIG_XIP_SPLIT_IMAGE.
       endif()
     endif()
 
+    # Targets for adding nrf70 wifi firmware patch. This workflow creates the nrf70_signed.hex and nrf70_update.bin files that can be
+    # used in DFU process, and multi-image DFU process.
+    if(CONFIG_NRF_WIFI_FW_PATCH_DFU)
+      set(nrf70_binary_name nrf70_update.bin)
+
+      sign(
+        SIGNED_BIN_FILE_IN ${PROJECT_BINARY_DIR}/nrf70.hex
+        SIGNED_HEX_FILE_NAME_PREFIX ${PROJECT_BINARY_DIR}/nrf70
+        SLOT_SIZE $<TARGET_PROPERTY:partition_manager,PM_NRF70_WIFI_FW_SIZE>
+        SIGNED_HEX_FILE_OUT nrf70_signed_hex
+        SIGNED_BIN_FILE_OUT nrf70_signed_bin
+        SIGNED_HEX_TEST_FILE_OUT nrf70_signed_test_hex
+        DEPENDS ${PROJECT_BINARY_DIR}/nrf70.hex
+      )
+
+      add_custom_target(mcuboot_nrf70_target
+        DEPENDS
+        ${nrf70_signed_hex}
+        ${nrf70_signed_bin}
+        ${nrf70_signed_test_hex}
+      )
+
+      set_property(GLOBAL PROPERTY
+        nrf70_wifi_fw_PM_HEX_FILE
+        ${nrf70_signed_hex}
+      )
+
+      set_property(GLOBAL PROPERTY
+        nrf70_wifi_fw_PM_TARGET
+        mcuboot_nrf70_target
+      )
+
+      add_dependencies(
+        mcuboot_sign_target
+        mcuboot_nrf70_target
+      )
+
+      # Add nrf70 update file to the existing bin files list before generating the zip file.
+      list(APPEND generate_bin_files
+        ${PROJECT_BINARY_DIR}/${nrf70_binary_name}
+      )
+
+      # Add nrf70-related sctipt params before generating the zip file.
+      list(APPEND generate_script_params
+        "${nrf70_binary_name}load_address=$<TARGET_PROPERTY:partition_manager,PM_NRF70_WIFI_FW_ADDRESS>"
+        "${nrf70_binary_name}image_index=2"
```

Comment:
Is this relevant for two images configuration (no netcore image)

Reply:
I've added configuration for the no net core image variant + configurations depending on XIP. Could you please verify that?